src/DbBackupsTracker.cpp

```diff
@@ -270,26 +270,34 @@ void DbBackupsTracker::setDataDbChangeNumber(int dataDbChangeNumber)
     emit dataDbChangeNumberChanged(this->dataDbChangeNumber);
 }
 
-bool DbBackupsTracker::isDbBackupChangeNumberGreater(int backupCCN, int backupDCN) const
+bool DbBackupsTracker::greaterThanWithWrapOver(int a, int b, int limit, int range) const
 {
-    // - the monitored file change number is higher than the one the device
-    bool result = (backupCCN > credentialsDbChangeNumber || backupDCN > dataDbChangeNumber);
+    bool res = (a > b) && ( (a - b) < range);
+    res = res | ((limit - b + a) < range);
```

Comment:
why no (a < b) &&  ((limit - b + a) < range); ?

Reply:
no need of (a < b), (limit -b + a) < range will be false any way, but if you want we could add the check.